### PR TITLE
More info on examples

### PR DESCRIPTION
**Pull request by suhasHere**

### CODE REVIEW

File: moq-arch.md [hunk 1]
```diff
@@ -2,87 +2,99 @@
...
+              │  ▮          pub │        ▮
+              │  ▮              │        ▮
+         ┌───────▮┐ ◀▬▮         │  ┌─────▮──┐
+     ┌──▶│ Relay-1│   ▮         └─▶│ Relay-2│◀▮▮
+     │   └─────┬──┘   ▮             ▲───┬───┘  ▮
+ pub │         │      ▮ sub     sub ▮   │      ▮ sub
+     │      pub│      ▮             ▮   │      ▮
+    ┌┴─────┐   │ ┌────▮─┐     ┌─────▮┐  │  ┌───▮──┐
+    │Alice │   └▶│ Bob  │     │ Carl │◀─┴─▶│Derek │
+    └──────┘     └──────┘     └──────┘ pub └──────┘
```
Comment (fluffy) [inline moq-arch.md:81]: I think the pub line from Relay 2 to Carl and Derek would be better we could show it as two lines. Love the bold lines. What tool did you use for this ?

File: moq-arch.md [hunk 2]
```diff
@@ -2,87 +2,99 @@
...
 
-In the simplest case, a web conferencing application could use a single
-relay to forward packets between users in a video conference. However a
-more typical scenario would have a delivery network made of multiple
-relays spread across several points of presence. QuicR is designed to
-make it easy to implement relays so that fail over could happen between
-relays with minimal impact to the clients and relays can redirect a
-client to a different relay.
+QuicR is pronounced something close to
+“quicker” but with more of a pirate "arrrr" at the end.
```
Comment (fluffy) [inline moq-arch.md:96]: There seems to be something wrong on the smart quoates on the arrr (like using unicode) - I thought I had fixed this in some other pull and now I am worried I did not merge that or something. Not a problem with this PR but I need to figure out where that went

File: moq-arch.md [hunk 3]
```diff
@@ -2,87 +2,99 @@
...
+
+The Relays are arranged in a logical tree (as shown below) where, 
+for a given application, there is an origin Relay at root of the tree 
+that controls the namespace. Publish messages are sent towards the 
+root of the tree and down the path of any subscribers to that 
+named object. QuicR is designed to make it easy to implement relays 
+so that fail over could happen between relays with minimal impact to 
+the clients and relays can redirect a client to a different relay.
+
+!--
```
Comment (fluffy) [inline moq-arch.md:62]: We need to check but I don't think we need the !-- a start and end of figure or at least they are not coming out how we might expect

File: moq-arch.md [hunk 4]
```diff
@@ -2,87 +2,99 @@
...
 systems such as messaging, gaming, and IoT.
 
-In the simplest case, a web conferencing application could use a single
-relay to forward packets between users in a video conference. However a
-more typical scenario would have a delivery network made of multiple
-relays spread across several points of presence. QuicR is designed to
-make it easy to implement relays so that fail over could happen between
-relays with minimal impact to the clients and relays can redirect a
-client to a different relay.
+QuicR is pronounced something close to
```
Comment (fluffy) [inline moq-arch.md:95]: I think it would make sense to move this paragraph into the end of the next "contributing" section

**Review by fluffy — APPROVED** [review]

File: moq-arch.md (hunk 2 above)
Comment (fluffy) [reply moq-arch.md:96]: I fixed this

File: moq-arch.md (hunk 4 above)
Comment (fluffy) [reply moq-arch.md:95]: I changed this

File: moq-arch.md (hunk 1 above)
Comment (suhasHere) [reply moq-arch.md:81]: Fixed

File: moq-arch.md (hunk 3 above)
Comment (suhasHere) [reply moq-arch.md:62]: this is fixed now

**Pull request merged**